drivers/video/Kconfig.mt9m114

```diff
@@ -6,8 +6,8 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
-menuconfig VIDEO_MT9M114
-        bool "MT9M114 Aptina CMOS digital image sensor"
-        depends on I2C
-        help
-          Enable driver for MT9M114 CMOS digital image sensor device.
+config VIDEO_MT9M114
```

Comment:
was this meant to go from menuconfig -> config? If so maybe doing that in a separate commit.

Reply:
There's a separate commit at https://github.com/zephyrproject-rtos/zephyr/pull/20165. Was an attempt to avoid a clash with it.

Reply:
Removed